polkadot/node/network/bridge/src/rx/mod.rs

```diff
@@ -1160,7 +1202,11 @@ async fn dispatch_validation_events_to_all<I>(
 	for event in events {
 		send_message!(event, StatementDistributionMessage);
 		send_message!(event, BitfieldDistributionMessage);
-		send_message!(event, ApprovalDistributionMessage);
+		if approval_voting_parallel_enabled {
```

Comment:
So, this is only temporary, right ? It feels a bit weird. Isn't there another way for this to work ? For example, what if we use approval-distribution as an entry point where we forward to approval-voting paralell or other just process the message normally (when parallel disabled)